src/de.rs

```diff
@@ -63,8 +76,6 @@ impl<'a, 'de> de::Deserializer<'de> for &'a mut Depythonizer<'de> {
             self.deserialize_unit(visitor)
         } else if obj.is_instance_of::<PyBool>()? {
             self.deserialize_bool(visitor)
-        } else if obj.is_instance_of::<PyByteArray>()? || obj.is_instance_of::<PyBytes>()? {
-            self.deserialize_bytes(visitor)
```

Comment:
I removed this because it was the quickest and easiest way to use the `on_error` callback when handling a `bytes` object. Obviously this breaks other use cases of this library, so a different solution will be needed here.